tracing-appender/src/lib.rs

```diff
@@ -99,7 +99,7 @@
 //!
 //! ```rust
 //! # fn docs() {
-//! let file_appender = tracing_appender::rolling::hourly("/some/directory", "prefix.log");
+//! let file_appender = tracing_appender::rolling::hourly("/some/directory", "prefix.log").unwrap();
```

Comment:
I'd prefer an `.expect("Unable create a rolling appender")` instead of an `.unwrap()`—we should show good panic hygine in documentation.